rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+
```

Comment:
- [ ] Let's also make sure we add attr_accessor for `:book, :person` you will need it when displaying on console later on 😇 so you can access person/book information through your rental